brigade-controller/cmd/brigade-controller/controller/controller_test.go

```diff
@@ -309,7 +309,7 @@ func TestController_NoSidecar(t *testing.T) {
 	}
 
 	c := pod.Spec.Containers[0]
-	if envlen := len(c.Env); envlen != 15 {
+	if envlen := len(c.Env); envlen != 16 {
 		t.Errorf("expected 15 items in Container.Env, got %d", envlen)
```

Comment:
"expected 15" should now be "expected 16"
maybe the 15 should be a constant somewhere in this file?)

Reply:
Yup adding a constant would be a good idea. I'll get that added later today.